src/main/java/com/google/sps/servlets/EnterQueue.java

```diff
@@ -61,15 +73,59 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             Key classKey = KeyFactory.stringToKey(classCode);
             Entity classEntity = datastore.get(txn, classKey);
 
-            ArrayList<String> updatedQueue = (ArrayList) classEntity.getProperty("studentQueue");
+            // Get date in mm/dd/yyyy format
+            DateTimeFormatter FOMATTER = DateTimeFormatter.ofPattern("MM/dd/yyyy");
+            LocalDate localDate = LocalDate.now(clock);
+            String currDate = FOMATTER.format(localDate);
```

Comment:
The Date type captures both time and date, but since we're only concerned with the number of visits per date, I think it would be more clear and easier to store only the calendar date.

Reply:
Yeah I think its a good idea to only capture the day. The issue with storing as a a `String` is that when we query visits we cant say "give me all the visits from the past week or month" If we store the date as a `Date` datastore will be able to do chronological queries. We could store with the current date and set the time always to midnight for example.